binaries/scootapi/main.go

```diff
@@ -1,26 +1,26 @@
 package main
 
 import (
+	"log"
+
 	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/scootdev/scoot/common/dialer"
 	"github.com/scootdev/scoot/scootapi/client"
-	"log"
 )
 
-// Command line tool to talk to CloudScoot Server
```

Comment:
Sure. Intent was preventing comments from being incomplete/out of date, but that is a premature optimization